wagtail_streamfield_migration_toolkit/migrate_operation.py

```diff
@@ -79,6 +80,13 @@ def deconstruct(self):
 
         return (self.__class__.__qualname__, args, kwargs)
 
+    @property
+    def migration_name_fragment(self):
+        fragments = OrderedDict(
+            (op.operation_name_fragment, None) for op, path in self.operations_and_block_paths
+        )
```

Comment:
```suggestion
        fragments = OrderedDict(
            (op.operation_name_fragment, None) for op, _ in self.operations_and_block_paths
        )
```

Suggestion: use `_` when a variable will be ignored, so the reader knows it's not important.

Using an `OrderedDict` is a concise way to express that we essentially want an ordered set, it would be good to add a short comment explaining this and why.